index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    
+    
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.1/css/all.min.css">
+    <title>Fork Me! FCC: Test Suite Template</title>
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div id="page-wrapper">
+      <header id="header">
+        <!-- edited by utsargo-->
+          <nav class="navbar navbar-expand-lg navbar-light bg-light justify-content-between">
+            <div id="logo">
+            <a class="navbar-brand" href="#">
+              <img id="header-img"src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="orginal trombones logo">
+            </a>
+          </div>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+              <span class="navbar-toggler-icon"></span>
+            </button>
+          
+            <div class="collapse navbar-collapse flex-row-reverse" id="navbarSupportedContent">
+              <ul class="navbar-nav ml-auto">
+                
+                <li class="nav-item">
+                  <a class="nav-link" href="#Features">Features</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#How_It_Works">How It Works</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#Pricing">Pricing</a>
+                </li>
+                
+                
+              </ul>
+              
+            </div>
+          </nav>
+       <!--edited by utsargo end-->
+      </header>
+      <div class="container" style="margin-top: 100px;">
+      <section id="middle">
+        <h2>Handcrafted, home-made masterpieces</h2>
+        <form id="form" action="https://www.freecodecamp.com/email-submit" class="d-flex flex-column align-items-center">
+          <input type="email" id="email" name="email" placeholder="Enter your email" required>
+          <input type="submit" name="" id="submit" value="Get Started" class="btn" style="background-color: #f1c40f; margin-top: 20px;">
+        </form>
+      </section>
```

Comment:
Great work so far, but next time I suggest you start talking like that:  `I think it would be better if...`, and always use emojis at the end of your comments to make them more friendly. 😄